solidity/test/support/helpers.js

```diff
@@ -214,6 +214,34 @@ export const toBuffer = uint8a => {
   return Buffer.from(uint8a)
 }
 
+export const splitRPCSignature = (oracleSignature) => {
+  let v = oracleSignature[64]
+  if (v < 27) {
+    v += 27
+  }
+  return {
+    v: v,
+    r: oracleSignature.slice(0, 32),
+    s: oracleSignature.slice(32, 64)
+  }
+}
+
+export const lPad = (string) => {
```

Comment:
I felt that `lPad` and `rPad` are an anti pattern I tried to avoid because they tie us conceptually to the hexadecimal string representation rather than the underlying data representation, I think a better way to do something like `lPad(0x00)`  is to use `toAddress(0)` or `toHash(0)` which are both semantically more obvious and use a more efficient underlying representation.